hack/make/.integration-daemon-start

```diff
@@ -98,8 +98,18 @@ if [ -z "$DOCKER_TEST_HOST" ]; then
 		)
 	fi
 
-	# "pwd" tricks to make sure $DEST is an absolute path, not a relative one
-	export DOCKER_HOST="unix://$(cd "$DEST" && pwd)/docker.sock"
+	if [ -n "${DOCKER_ROOTLESS}" ]; then
+		# "pwd" tricks to make sure $DEST is an absolute path, not a relative one
+		export DOCKER_HOST="unix://$(cd "$DEST" && pwd)/docker.sock"
+	else
+		# Put socket in /run because:
+		# 1. That's the normal place for such things
+		# 2. When running on Docker For Mac, if you need to run tests with the bundles dir mounted (e.g. to poke through test artifacts).
+		#    the socket will not work because it will be over osxfs.
+		mkdir -p /run/docker
+		local sock_dir=$(mktemp -d -p /run/docker)
```

Comment:
hm, looking now; `local` is non-POSIX

Reply:
oh, nevermind looks like the script is using `bash` so should be ok

Reply:
We can't use `local` anyway because this is not a shell function.

Reply:
still wondering if we need the wrapper directory (which will also be the same as /var/run/docker)?; We could just store sockets in /run/, and name them `docker-<daemon id>.sock`

Reply:
oh, nevermind I guess this PR only did the main daemon, not the ones we start during tests (those had the same issue FWIW)